bootstrapping_tools/bootstrapping.py

```diff
@@ -26,6 +48,15 @@ def lambda_calculator(
 
 
 def remove_distribution_outliers(data, number_of_std=2.698):
+    """This function removes outliers from distribution using the standard deviation and a range
```

Comment:
Creo que el _"This function"_ sale sobrando:

```suggestion
    """Remove outliers from distribution using the standard deviation and a range
```